page.php

```diff
@@ -1,5 +1,32 @@
 <?php
 get_header();
+
+function render_submenu($post) {
```

Comment:
it only happens on that 1 template so i figured better to declare it there rather than fill up functions